python/benchmarks/plasma.py

```diff
@@ -0,0 +1,68 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+import numpy as np
+
+import pyarrow as pa
+import pyarrow.plasma as plasma
+
+from . import common
+
+
+class PlasmaBenchmarks(object):
```

Comment:
From a high-level point of view, what do we want to benchmark here? The plasma client, or client + store? You may want to choose an explicit timer that reflects that decision (see the `timer` attribute [here](https://asv.readthedocs.io/en/latest/writing_benchmarks.html#timing)). Also add a docstring :-)

Reply:
I added a docstring, not sure what you mean with the timer attribute; the default seems fine to me :)

Reply:
I mean that if you want to time the client overhead, you need a timer that measures the elapsed CPU time of the current process is required (but that is `asv`'s default, AFAICT). If OTOH you want to measure the whole client + server roundtrip, you need a timer that measures the elapsed wall clock time.

Reply:
Oh I see, thanks for pointing that out. I do want to measure the roundtrip wallclock time, and think timeit which is the default for asv is doing that (according to https://docs.python.org/3.0/library/timeit.html).

Reply:
Actually, according to https://asv.readthedocs.io/en/latest/writing_benchmarks.html#timing the default is process CPU time (by way of `time.process_time`). Which makes sense in most cases but not here :-)

Reply:
Thanks, I didn't realize they were overriding the timeit default, should be fixed now :)